csharp/ql/test/utils/model-generator/theorems/CaptureTheoremsForFreeSummaryModels.expected

```diff
@@ -0,0 +1,14 @@
+| Summaries;CollectionTheorems1<>;false;Add;(T);;Argument[0];Argument[this].SyntheticField[ArgT].Element;taint;generated |
```

Comment:
I think all of these generated steps should be value steps rather than taint steps.